src/ember-app.js

```diff
@@ -310,7 +310,10 @@ function createShoebox(doc, fastbootInfo) {
     if (!shoebox.hasOwnProperty(key)) { continue; }
 
     let value = shoebox[key];
-    let scriptText = doc.createTextNode(JSON.stringify(value));
+    let textValue = JSON.stringify(value);
+    textValue = textValue.replace(/<\/(script[ />])/i, '<\\/$1');
```

Comment:
Doesn't this need a `g` flag? I think this will only catch the first instance of `</script`.